fix/stuckdoors.lua

```diff
@@ -0,0 +1,120 @@
+-- Fix doors that are frozen in 'open' state.
+-- This may happen after people mess with the game by (incorrectly) teleporting units or items
+-- A door may stick open if the map occupancy flags are wrong.
+
+--[====[
+
+    fix/stuckdoors
+    ===============
+    Fix doors that are stuck open due to incorrect map occupancy flags, eg due to
+    incorrect use of `teleport`.
+
+]====]
+
+if #{...} > 0 then
+    qerror('This script takes no arguments.')
```

Comment:
instead of throwing an error, consider `print(dfhack.script_help())`. That outputs the rendered help text from the associated .rst file.

Reply:
that looks good. Thanks!